qml/pages/ProgramDelegate.qml

```diff
@@ -6,7 +6,8 @@ import "../js/yleApi.js" as YleApi
 ListItem {
     id: listItem
     contentWidth: ListView.view.width
-    contentHeight: Theme.itemSizeLarge
+    //contentHeight: Theme.itemSizeLarge
```

Comment:
Does this predefined size look bad on some device? Or are the changes only because of the text spacing?

Reply:
It's because of the text sizing; the content is either not centered properly or overflows (at least when using a different system font size). I haven't found more elegant way to do it, so I used this method. The commented code is indeed an oversight, I'll clean it up.